package.json

```diff
@@ -109,8 +109,8 @@
 		"shallow-equal": "1.0.0",
 		"shallow-equals": "1.0.0",
 		"shallowequal": "1.1.0",
-		"sprintf-js": "1.1.1",
 		"source-map-loader": "0.2.3",
+		"sprintf-js": "1.1.1",
 		"stylelint-config-wordpress": "13.1.0",
 		"uuid": "3.3.2",
 		"webpack-bundle-analyzer": "3.0.2",
```

Comment:
After the change here, I expect at least a small change in the `package-lock.json` file. (That's why the tests are failing). You should run `npm install` and commit the package lock.

Reply:
Hum ok. I understand now why package-lock.json is needed :) 
I just didn't commit this file. I'm going to do it.